src/app/3d/qgs3dmapconfigwidget.cpp

```diff
@@ -114,6 +114,7 @@ Qgs3DMapConfigWidget::Qgs3DMapConfigWidget( Qgs3DMapSettings *map, QgsMapCanvas
 
   widgetLights->setPointLights( mMap->pointLights() );
   widgetLights->setDirectionalLights( mMap->directionalLights() );
+  widgetLights->setOpenLightTab( mMap->lightsWidgetTab() );
```

Comment:
A better approach here is to store/retrieve  the last used tab in QSettings, rather then in the map settings. That avoids the "bleed" of application/GUI related settings into the core 3d classes which is done here (the addition of lastOpenLightTab/setOpenLightTab in the base 3d map settings class).